src/ui/layouts/common/layouts/AuthenticatedLayout/AuthenticatedSidebar/Menu/index.tsx

```diff
@@ -19,9 +19,9 @@ export const Menu: React.FC = () => {
         text={translate('menu.home.text')}
         exact
       />
-      <Box marginVertical="md">
+      {/* <Box marginVertical="md">
         <Separator.Light />
-      </Box>
+      </Box> */}
```

Comment:
Is there a specific reason why we left this commented but didn't remove it?

Reply:
I left the commented code just for reference because the code base is new for us. We will surely remove the dead code.

Reply:
Can we add like a TODO comment here just so that we remember and don't forget?
Thanks

Reply:
fixed